Tools/CISupport/ews-build/steps.py

```diff
@@ -53,9 +53,11 @@
 BUG_SERVER_URL = 'https://bugs.webkit.org/'
 COMMITS_INFO_URL = 'https://commits.webkit.org/'
 S3URL = 'https://s3-us-west-2.amazonaws.com/'
+S3_BUCKET = f'ews-archives.webkit{custom_suffix}.org'
 S3_RESULTS_URL = f'https://ews-build{custom_suffix}.s3-us-west-2.amazonaws.com/'
 CURRENT_HOSTNAME = socket.gethostname().strip()
-EWS_BUILD_HOSTNAME = 'ews-build.webkit.org'
+EWS_BUILD_HOSTNAMES = ['ews-build.webkit.org', 'ews-build']
+EWS_BUILD_UAT_HOSTNAMES = ['ews-build.webkit-uat.org', 'ews-build-uat']
```

Comment:
Please add ews-build.webkit-dev.org and ews-build-dev to this list. You might want to change the name of this variable to "nonprod" or something. Otherwise, create another DEV list of hostnames and modify the conditions used in other places accordingly.